circle.yml

```diff
@@ -21,16 +21,18 @@ test:
   post:
     - cp -r frontend/coverage $CIRCLE_ARTIFACTS/
     - cp server/lint-results.html $CIRCLE_ARTIFACTS
+    - mkdir $CIRCLE_ARTIFACTS/server-coverage
+    - cp -r server/coverage $CIRCLE_ARTIFACTS/server-coverage/
 deployment:
   prod:
     branch: master
     owner: 18F
     commands:
-      - cd frontend; yarn run build --env=prod;
+      - cd frontend; yarn run build;
       - ./cg-deploy/deploy.sh fs-intake-prod
   staging:
-    branch: sprint-5-development
```

Comment:
I think this needs to be updated from sprint-5-development. We should be deploying on sprint-5-development